src/regtech_cleanup_api/entities/repos/filing_repo.py

```diff
@@ -16,16 +16,24 @@
 T = TypeVar("T")
 
 
-def get_user_action_ids(session: Session, lei: str = None, period_code: str = None):
-    filing = submission_repo.get_filing(session, lei, period_code)
+def get_user_action_ids(
+    session: Session,
+    lei: str = None,
+    period_code: str = None,
+    submissions: bool = False,
+):
+    filing_user_action_id = []
+    if not submissions:
+        filing = submission_repo.get_filing(session, lei, period_code)
+        filing_user_action_id = (
+            [filing.creator_id] if filing.creator_id is not None else []
```

Comment:
You don't need to check for is not None.  Python by default will check for None and consider it False.  So [filing.creator_id] if filing.creator_id else [] should work.